src/bun.js/builtins/js/ProcessObjectInternals.js

```diff
@@ -450,6 +453,17 @@ function getStdinStream(fd_, rawRequire, Bun) {
       return fd_;
     }
 
+    get isRaw() {
+      return (this.#internalIsRaw ??=
+        require("node:tty")[Symbol.for("__BUN_INTERNAL_DO_NOT_USE_ELSE_RISK_TERMINATION__isRaw")])(fd_);
```

Comment:
We can use Bun.lazy for this instead of an export

Reply:
Oh right, I forgot about that. Good call